docs/UserGuide.md

```diff
@@ -93,11 +103,14 @@ A person can have any number of tags (including 0)
 
 Examples:
 * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 b/11 09 2001`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 b/11 09 2001 t/criminal`
+* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/91234567 b/11 09 2001 t/criminal`
+
+**Note:** As of version 1.3, only valid email addresses and Singapore phone numbers are accepted. 
+Birthdays should be added in DD MM YYYY format.
 
 ### Listing all persons : `list`
 
-Shows a list of all persons in the FART book.
+Shows a list of all persons in the FART book. The list is displayed in chronological order (i.e., the order you added the contacts in).
```

Comment:
Can I kindly confirm that list is/should be displayed in chronological order, and not alphabetical order?